anyway/core/utils.py

```diff
@@ -0,0 +1,19 @@
+class Utils():
```

Comment:
What is the difference between `utils` and `utilities`?

Reply:
```utilities.py``` will be removed, and all the necessary functions will be moved to ```utils``` file inside the ```Utils``` class